rmgpy/tools/canteramodel.py

```diff
@@ -121,7 +149,7 @@ def __str__(self):
         return string
 
 
-def generate_cantera_conditions(reactor_type_list, reaction_time_list, mol_frac_list, Tlist=None, Plist=None, Vlist=None):
+def generate_cantera_conditions(reactor_type_list, reaction_time_list, mol_frac_list, surface_mol_frac_list=[], Tlist=None, Plist=None, Vlist=None):
```

Comment:
Can you default this to None and initialize the list within the function? It's best practice not to use mutable defaults.